material-ui/material-ui.scss

```diff
@@ -120,3 +121,120 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  input {
+    position: relative;
+    left: 30px;
+    width: 40px;
+    height: 40px;
+    margin: 0;
+    padding: 20px;
+    transform: scale(1);
+    transition: opacity .3s, transform .2s;
```

Comment:
Do you mind explaining the goal of this `transition`?

Reply:
This transition is to give an organic effect on the circle background around the checkbox when clicked or unclicked.

Reply:
Agree @Yairsaenz7 the effect is very nice. I should probably add the same for the radio input as well then...